conf/roslin.config

```diff
@@ -34,19 +34,15 @@ process {
     maxErrors = '-1'
     maxRetries = 3
 
+    // Force singularity version 3 as version 4 cause FUSE/squashfs issues on Eddie
     beforeScript =
     """
     . /etc/profile.d/modules.sh
-    module load singularity
+    module load igmm/apps/singularity/3
```

Comment:
I'd strongly recommend trying to resolve this, the last 3 release is almost two years old and there's been a steady stream of fixes since.

Reply:
We have updated our nodes very recently. 
The IS are working on this issue. 
While waiting for a resolution, non nextflow power user struggle to debug this singularity. 
I'll switch back as soon as singularity 4 is fixed ;)